src/pages/Lend/Filter/SelectedRefinements.vue

```diff
@@ -1,38 +1,45 @@
 <template>
 	<div class="selected-refinements">
-		<ais-current-refinements
-			:transform-items="transformItems"
-		>
+		<ais-current-refinements :transform-items="transformItems">
 			<div slot-scope="{ items }">
-				<div class="filter-summary-container">
-					<div>{{ items.length }} Filters Applied</div>
-					<div v-if="isCollapsible" class="show-toggle-container">
-						<div v-if="isCollapsed" @click="handleClickShowMore">Show</div>
-						<div v-else @click="handleClickShowFewer">Hide</div>
+				<div class="row align-middle">
+					<div :class="[ isCollapsed ? 'medium-10' : 'medium-12', 'small-12 columns' ]">
```

Comment:
I don't think this approach (changing the number of columns based on `isCollapsed`) is working. Maybe it's worth trying another approach?